frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -345,7 +350,7 @@ pub mod pallet {
 
                 consumed_weight + T::DbWeight::get().reads_writes(5, 3)
             } else {
-                T::DbWeight::get().reads(4)
+                return T::WeightInfo::on_initialize_not_new_era();
```

Comment:
Sorry if we misunderstood each other - just to repeat what we discussed on sync - we either go with fully-benchmarked solution or just go with _manual_ calculation.

Imo, you should drop the benchmarks :P

Reply:
We're on the same page! I don't think I had pushed the updates for this yet.